server/centralized.go

```diff
@@ -226,6 +236,16 @@ func (c *Centralized) batchReads() {
 				c.log.Trace.Printf("Read: add to batch, size=%v\n", len(c.ReadBatch))
 			}
 			continue
+		case <-tick:
+			if len(c.ReadBatch) > 0 {
+				for len(c.ReadBatch) < config.ReadBatch {
```

Comment:
I see. Seems like it would be reasonable to do this at shard, minimize how much complexity is  exposed